### PR TITLE
#50/test/addTestsForRangeComponent

### LINKED ISSUE #50: (SP: 1) Write tests for range component
Writing tests don't forget to mock elements if neccessary.

Test cases:

- [x] it should renders correctly
- [x] it should call onChange when slider is moved
- [x] it should call onChange when input is changed
- [x] it should not call onChange when input is changed wit not a number
- [x] it should call onChange whith min number if input is empty
- [ ] it should update prices when input is blurred and input is greater than max value
- [x] it should not update prices when input is blurred and value in input has not changed

### PR DESCRIPTION
**Pull request by aleksandraostrovskaya**
Tests for the AppRange component have been written.

## Summary by CodeRabbit

- **Tests**
	- Added comprehensive unit tests for the `AppRange` component
	- Verified component rendering, slider interactions, and input handling
	- Tested various scenarios including edge cases for input validation and state management

### CODE REVIEW

**Review by ANOVIKOVA21 — APPROVED** [review]

**Review by ZerkalAndrii — APPROVED** [review]

**Review by irynalaitaruk — APPROVED** [review]

**Review by 2024JS — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

File: src/tests/unit/components/app-range/AppRange.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react'
+import { describe, vi } from 'vitest'
```
Comment (vlad-khrychov) [inline src/tests/unit/components/app-range/AppRange.spec.jsx:2]:
```suggestion
import { vi } from 'vitest'
```

File: src/tests/unit/components/app-range/AppRange.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+describe('AppRange Component', () => {
+  const mockOnChange = vi.fn()
+
+  beforeEach(() => {
+    mockOnChange.mockClear()
+  })
+
+  it('should render correctly', () => {
+    render(<AppRange max={100} min={0} onChange={mockOnChange} />)
```
Comment (vlad-khrychov) [inline src/tests/unit/components/app-range/AppRange.spec.jsx:13]: You can move it in `beforeEach`
